src/pages/test-page.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import { Layout } from "../components/Layout/Layout"
+import SettingsModal from "../components/SettingsModal/SettingsModal"
+
+const TestPage = () => {
+  const [modalActive, setModalActive] = useState<boolean>(false)
+
+  const toggleModal = () => {
+    // eslint-disable-next-line no-console
+    console.log("toggle")
```

Comment:
I know this is only for testing, but before we merge we should remove the ignore and the `console.log` 😄